training_config/ner.jsonnet

```diff
@@ -58,8 +58,7 @@
         "bidirectional": true
     },
   },
-  "iterator": {
-    "type": "basic",
+  "data_loader": {
     "batch_size": 64
```

Comment:
Isn't the batch sampler necessary here?

Reply:
No, because the data loader can take a batch size directly